services/api-server/src/simcore_service_api_server/api/routes/solvers_jobs.py

```diff
@@ -62,7 +62,7 @@ def _compose_job_resource_name(solver_key, solver_version, job_id) -> str:
 )
 async def list_jobs(
     solver_key: SolverKeyId,
-    version: str,
+    version: VersionStr,
```

Comment:
This changes the API specs. Please update version (patch) and  ``make openapi.json`` in this project

Reply:
ok then why was it like this for only 2 of the API calls?
I will upgrade the version